components/webhook-mgt/org.wso2.carbon.identity.webhook.metadata/src/main/java/org/wso2/carbon/identity/webhook/metadata/api/model/EventProfileMetadata.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.webhook.metadata.api.model;
+
+/**
+ * Represents a webhook event profile metadata.
+ */
+public class EventProfileMetadata {
+
+    private String name;
+    private String uri;
+
+    /**
+     * Default constructor.
+     */
+    public EventProfileMetadata() {
+
+    }
```

Comment:
Is this needed ? I think we don't need to have this default constructor as name and uri is a must for the profile